planning/behavior_path_planner/src/scene_module/start_planner/start_planner_module.cpp

```diff
@@ -278,7 +259,7 @@ BehaviorModuleOutput StartPlannerModule::plan()
   PathWithLaneId path;
 
   // Check if backward motion is finished
-  if (status_.driving_forward) {
+  if (status_.driving_forward || status_.backward_driving_complete) {
```

Comment:
workaround

Reply:
status_.driving_forward should be true after finishing backward driving but it is still false.
something wrong